test/e2e/config/crossbrowser.conf.js

```diff
@@ -43,36 +43,78 @@ const config = {
       browserName: 'chrome',
       version: 'latest',
       platform: 'Windows 10',
-      name: 'chrome-tests',
+      name: 'chrome-win-tests',
       tunnelIdentifier: 'reformtunnel',
       extendedDebugging: true,
       sharedTestFiles: false,
       maxInstances: 1
     },
 
+    {
+      browserName: 'firefox',
+      version: 'latest',
+      platform: 'Windows 10',
+      name: 'firefox-win-tests',
+      tunnelIdentifier: 'reformtunnel',
+      extendedDebugging: true,
+      sharedTestFiles: false,
+      maxInstances: 1
+    },
+
+    {
+      browserName: 'internet explorer',
+      platform: 'Windows 10',
+      version: 'latest',
+      name: 'IE-TEST',
+      tunnelIdentifier: 'reformtunnel',
+      extendedDebugging: true,
+      sharedTestFiles: false,
+      maxInstances: 1
+    },
 
     {
       browserName: 'safari',
-      platform: 'macOS 10.13',
-      version: '11',
+      platform: 'macOS 10.15',
+      version: '13.1',
       name: 'Safari-TEST',
       tunnelIdentifier: 'reformtunnel',
       extendedDebugging: true,
       sharedTestFiles: false,
       maxInstances: 1
-
     },
 
     {
       browserName: 'MicrosoftEdge',
-      version: '18.17763',
-      platform: 'Windows 10',
+      platform: 'macOS 10.15',
```

Comment:
Why has this been changed to macOS? Curious...

Reply:
this is all crossbrowser config as per Saucelabs